examples/tv-casting-app/android/App/app/src/main/jni/cpp/MatterCallbackHandler-JNI.h

```diff
@@ -104,6 +104,20 @@ template <typename T>
 SuccessHandlerJNI<T>::~SuccessHandlerJNI(){};
 
 // COMMISSIONING AND CONNECTION
+class SessionEstablishmentStartedHandlerJNI : public SuccessHandlerJNI<void *>
+{
+public:
+    SessionEstablishmentStartedHandlerJNI() : SuccessHandlerJNI("(Ljava/lang/Object;)V") {}
+    jobject ConvertToJObject(void * responseData) { return nullptr; }
+};
+
+class SessionEstablishedHandlerJNI : public SuccessHandlerJNI<void *>
+{
+public:
+    SessionEstablishedHandlerJNI() : SuccessHandlerJNI("(Ljava/lang/Object;)V") {}
+    jobject ConvertToJObject(void * responseData) { return nullptr; }
```

Comment:
why do we return nullptr here?

Can you add documentation to explain?

Reply:
This success callback is called without passing any value params. Its Java counterpart extends SuccessCallback\<Void> and its handle() expects a Void param. So it makes sense to just pass in nullptr here. Added docs to explain that.